addons/storyshots/README.md

```diff
@@ -40,11 +40,19 @@ Usually, you might already have completed this step. If not, here are some resou
 
 ### Configure Jest for React
 StoryShots addon for React is dependent on [react-test-renderer](https://github.com/facebook/react/tree/master/packages/react-test-renderer), but 
-doesn't install it, so you need yo install it separately.
+doesn't install it, so you need to install it separately (read [here](#deps-issue) why).
```

Comment:
It's a super-nit, but I prefer when text keeps making sense if you replace links with their text, that's why I don't really like links with "here" text.

Can we just make "doesn't" a link?